app/retail/views.py

```diff
@@ -245,9 +245,18 @@ def how_it_works(request, work_type):
     """Show How it Works / Funder page."""
     if work_type not in ['funder', 'contributor']:
         raise Http404
-
+    title = ''
```

Comment:
Since we're validating the `work_type` is in our list range, we can drop the `title = ''` | `desc = ''` assignments.